### PR TITLE
fix(dropdown): Adding inverted style, updating colors for Teams theme

### PR DESCRIPTION
**Pull request by notandrew**
* adding inverted style to dropdown
* updating dropdown colors
* fixing high contrast colors
* updating the selected item colors
* updating focus styles

image

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/12272)

### CODE REVIEW

File: packages/fluentui/react/src/themes/teams-high-contrast/componentStyles.ts [hunk 1]
```diff
@@ -1,2 +1 @@
 export { default as MenuItem } from './components/Menu/menuItemStyles';
-export { default as Dropdown } from './components/Dropdown/dropdownStyles';
```
Comment (layershifter) [inline packages/fluentui/react/src/themes/teams-high-contrast/componentStyles.ts]: Should be `dropdownStyles.ts` removed at all for High contrast?

File: packages/fluentui/react/src/themes/teams/components/Dropdown/dropdownSearchInputStyles.ts [hunk 2]
```diff
@@ -1,17 +1,22 @@
...
     flexBasis: v.comboboxFlexBasis,
     flexGrow: 1
   }),
 
-  input: ({ props: p }): ICSSInJSStyle => ({
+  input: ({ props: p, theme: { siteVariables } }): ICSSInJSStyle => ({
     width: '100%',
     backgroundColor: 'transparent',
     borderWidth: 0,
+    ':focus-within': {
```
Comment (layershifter) [inline packages/fluentui/react/src/themes/teams/components/Dropdown/dropdownSearchInputStyles.ts]: FYI: This will not work in IE and probably old Edge

Comment (notandrew) [reply packages/fluentui/react/src/themes/teams/components/Dropdown/dropdownSearchInputStyles.ts]: It was extra anyway. Removing it.

File: packages/fluentui/react/src/themes/teams-high-contrast/componentStyles.ts (hunk 1 above)
Comment (layershifter) [reply packages/fluentui/react/src/themes/teams-high-contrast/componentStyles.ts]: FYI: https://github.com/OfficeDev/office-ui-fabric-react/pull/12278 I merged these changes, so please rebase your branch 🙏

Comment (notandrew) [reply packages/fluentui/react/src/themes/teams-high-contrast/componentStyles.ts]: I'll wait until the big comma/space change coming soon...

**Comment by notandrew** [comment]
Due to the large amount of differences between this & the current repo, I'm closing this and opening this:
https://github.com/microsoft/fluentui/pull/12397

**Pull request closed without merging**